script.js

```diff
@@ -46,41 +46,48 @@ var state = {
     questionIndex: 0 
 };
 
-//What can we DO to the state?
-//How should we render the Data to the DOM?
+var resetCurrentScore = function() {
+    state.score = 0;
+};
+var increaseQuestionIndex = function() {
+    state.questionIndex++;
+};
+var increaseStateScore = function() {
+    state.score++;
+};
 
-var setQuestion = function() { //takes state and questionIndex
+var setQuestion = function() { 
     console.log('question set');
     console.log(state.questionIndex);
-    var question = QUESTIONS[state.questionIndex]; //stores the questionIndex from QUESTIONS in question
```

Comment:
You should stay away from all caps. it is specifically used when calling a NAMESPACE. So maybe `questions[]`.